### DISCUSSION TITLE
Invoke-RestMethod parameter old and therefore blocked by some Customers Firewalls

### DISCUSSION THREAD

**Discussion by AndreDorsch** [EUCPilots/evergreen-module#659] [Q&A]
Hi got trouble that the EG was blocked by zscaler as the cromium versiondetectet was 122 ;-(

Check with VERBOSE: 

VERBOSE: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: UserAgent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) **Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0..**

GET https://versionhistory.googleapis.com/v1/chrome/platforms/win64/channels/extended/versions/all/releases with 0-byte payload
WARNING: Invoke-EvergreenRestMethod: The remote server returned an error: (403) Forbidden., with: https://versionhistory.googleapis.com/v1/chrome/platforms/win64/channels/extended/versions/all/releases.

Finding in the config "WindowsPowerShell\Modules\Evergreen\2404.928\Evergreen.json" you give the UserAgent parameters with Versions 122

  "UserAgent": {
		"Base": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.",
        "Download": [
            "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.",
            "Googlebot/2.1 (+http://www.google.com/bot.html)"
        ]

Can you update your parameters in next EG Releases please?

is it possible to overwrite this central UserAgent Parameters in a custormers EG Skript?

BR Andre

**Comment by aaronparker** [comment]
I'll update the default user agent used, and make sure to keep it current.

I should be able to add on option for a customer user agent in a future release.
